documentation/faq/index.adoc

```diff
@@ -40,14 +40,24 @@ Where can I find answers about the Browser Simulator?::
 
 Where can I find answers about the CordovaSim?::
   See the link:./cordovasim.html[CordovaSim FAQ]
-  
+
 Where can I find answers about the Visual Page Editor?::
   See the link:./visualeditor.html[Visual Page Editor FAQ]
-  
+
 === Server/Publishing
 
 [qanda]
 I use eclipse with the jboss tools installed, is there any way to republish my app that doesnt involve what feels like 50 clicks by opening server tab, expanding server, right clicking the module and selecting full publish?::
 
 Use menu:Help[Key assist] and a yellow window pops up with current keyboard shortcuts. Type "Full" and a "Full Publish module" should show up with the right key.
 It is a bit long, but you can reconfigure it if you wish to under menu:Preferences[General, Keys].
+
+=== LiveReload
+
+[qanda]
+I use JBoss Tools with LiveReload enabled, but my HTML Page is not refreshed after page a change was published on the application server. Why ?::
```

Comment:
Original: I use JBoss Tools with LiveReload enabled, but my HTML Page is not refreshed after page a change was published on the application server. Why ?

Suggested: I use JBoss Tools with LiveReload enabled, but my HTML Page did not refresh as expected after I published a change on the application server.